src/Http/Controllers/ResetPasswordController.php

```diff
@@ -3,16 +3,15 @@
 namespace Statamic\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
```

Comment:
You may want to remove this usage statement. This file doesn't use the `Hash` facade, it's also causing an issue with Style CI: https://github.styleci.io/analyses/2QywDg?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.